ci/array-api-skips.txt

```diff
@@ -1,104 +1 @@
-array_api_tests/test_signatures.py::test_func_signature[greater]
-array_api_tests/test_signatures.py::test_func_signature[greater_equal]
-array_api_tests/test_signatures.py::test_func_signature[isfinite]
-array_api_tests/test_signatures.py::test_func_signature[isinf]
-array_api_tests/test_signatures.py::test_func_signature[isnan]
-array_api_tests/test_signatures.py::test_func_signature[less]
-array_api_tests/test_signatures.py::test_func_signature[less_equal]
-array_api_tests/test_signatures.py::test_func_signature[logaddexp]
-array_api_tests/test_signatures.py::test_func_signature[matrix_transpose]
-array_api_tests/test_signatures.py::test_func_signature[vecdot]
-array_api_tests/test_signatures.py::test_func_signature[take]
-array_api_tests/test_signatures.py::test_func_signature[trunc]
-array_api_tests/test_signatures.py::test_func_signature[square]
-array_api_tests/test_signatures.py::test_func_signature[remainder]
-array_api_tests/test_signatures.py::test_func_signature[argmax]
-array_api_tests/test_signatures.py::test_func_signature[argmin]
-array_api_tests/test_signatures.py::test_func_signature[from_dlpack]
-
-# linalg namespace
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.cross]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.matmul]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.cholesky]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.matrix_norm]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.matrix_rank]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.matrix_transpose]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.outer]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.pinv]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.svdvals]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.tensordot]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.vecdot]
-array_api_tests/test_signatures.py::test_extension_func_signature[linalg.vector_norm]
-
-# Array object namespace
-array_api_tests/test_signatures.py::test_array_method_signature[__bool__]
-array_api_tests/test_signatures.py::test_array_method_signature[__complex__]
-array_api_tests/test_signatures.py::test_array_method_signature[__dlpack__]
-array_api_tests/test_signatures.py::test_array_method_signature[__dlpack_device__]
-array_api_tests/test_signatures.py::test_array_method_signature[__float__]
-array_api_tests/test_signatures.py::test_array_method_signature[__index__]
-array_api_tests/test_signatures.py::test_array_method_signature[__int__]
-array_api_tests/test_signatures.py::test_array_method_signature[__setitem__]
-
-# missing 'stream' from signature
-array_api_tests/test_signatures.py::test_array_method_signature[to_device]
+array_api_tests/test_searching_functions.py::test_nonzero_zerodim_error
```

Comment:
Why not also have this in the xfails file?

Reply:
It's a bit tricky: 
```
array_api_tests/test_searching_functions.py::test_nonzero_zerodim_error
```
now passes as it just expects an exception to happen. But in `xfails` file we have:
```
array_api_tests/test_searching_functions.py::test_nonzero
```
as `nonzero` still has some discrepancies. This line expects XFAIL from both `test_nonzero` and `test_nonzero_zerodim_error` (because `test_nonzero` is a prefix of `test_nonzero_zerodim_error`!) but `test_nonzero_zerodim_error` passes, causing unexpected XPASS failure.

It's caused by the fact that one test's name is exactly a prefix of another one, where one passes and other doesn't - I would say bad luck.

Therefore let's keep `test_nonzero_zerodim_error` in skips until we have `test_nonzero` passing. After that we can remove skips file.

Reply:
Thankfully that's the only test (I think) where the name is exactly a prefix of another test's name 😅 

Prefixes are useful in `skips`/`xfails` files to include a whole group of tests/test files.